client/src/styles/history-chat.css

```diff
@@ -38,6 +38,8 @@ ul li p {
 
 .visible {
   display: block;
+  animation: fadeInAnimation ease 0.3s;

```

Comment:
probably you don't want an `ease` on a fade animation; `ease` is for motion...and make this make `0.15s`

Reply:
ease removed, and it still looks great without ease